### PR TITLE
Add cve-feed-osv project

### PR DESCRIPTION
**Pull request by MadhavJivrajani**
Ref https://github.com/kubernetes/org/issues/4873

/hold

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -2815,6 +2815,7 @@ sigs:
     contact:
       slack: sig-security-tooling
     owners:
+    - https://raw.githubusercontent.com/kubernetes-sigs/cve-feed-osv/main/OWNERS
```
Comment (MadhavJivrajani) [inline sigs.yaml:2818]: @PushkarJ can you confirm if adding it as a project under the `sig-security-tooling` subproject is accurate or do we require a new subproject to be created?

Comment (PushkarJ) [reply sigs.yaml:2818]: Is this mostly for documentation purposes or does this mean some permissions like approving or lgtm will be granted to owners of that subproject?

Comment (PushkarJ) [reply sigs.yaml:2818]: If it's documentation purposes it's fine to keep the sub-project as sig-security-tooling

**Pull request merged**